src/components/validationStatus.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div class="validation-bar-container">
+    <div class="validation-container" v-if="toggleValidationPanel">
+      <span class="validation-container__defaultMessage" v-if="!numberOfValidationErrors">no problems to report</span>
+      <div class="validation-container__list"  v-for="error in errorList" :key="error.id">
+        <span class="validation-container__list--id">{{ error.id }}</span>
+        <span class="validation-container__list--message">
+          <span class="validation-container__list--key">{{ error.errorKey }}</span>
+          <span>{{ error.message }}</span>
+        </span>
+        <span class="validation-container__list--errorCategory"
```

Comment:
Can we make this an icon and smaller  on the left side like a bullet.  Use these icons:

Error: https://fontawesome.com/icons/times-circle?style=solid
Warning: https://fontawesome.com/icons/exclamation-triangle?style=solid